doc/wolfssl/README

```diff
@@ -115,8 +118,8 @@ The following line in the lighttpd.conf will specify the lighttpd log file:
 Prior to the log being opened, log messages will be directed to stderr.
 
 To use lighttpd logging the following function is called with a lighttpd
-server *srv arguement:
-    log_error_write(srv, __FILE__, __LINE__, "ss", "", "desired message");
+server *srv argument:
+    log_error_write(srv, __FILE__, __LINE__, "s", "", "desired message");
```

Comment:
The new code is incorrect and the last argument is unused.  Please remove this code example from the documentation.  log_error_write() is part of the lighttpd basic interfaces and is not specific to mod_openssl or wolfSSL.